examples/howto/Model_Builder.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>Add:</p><p>Notice that <code>load()</code> is a class-method, so we call it on the class, not the object. The benefit of that is, that when you <code>load()</code> the model in a different environment, all you need is access to the model class (e.g. in a <code>.py</code> file) and your <code>.nc</code> <code>InferenceData</code> file -- the data and model configuration are all saved and loaded.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>